source/reference/api/start.txt

```diff
@@ -151,12 +225,103 @@ Request
 Response
 ~~~~~~~~
 
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+   :copyable: false
+
+Example 3 - Start Sync from Replica Set to Sharded Cluster
+----------------------------------------------------------
+
+Request
+~~~~~~~
+
+.. literalinclude:: /includes/api/requests/start-rs-shard.sh
+   :language: shell
+
+Response
+~~~~~~~~
+
 .. literalinclude:: /includes/api/responses/success.json
    :language: json
    :copyable: false
 
 Behavior
 --------
 
+State
+~~~~~
+
 If the ``start`` request is successful, ``mongosync`` enters the
 ``RUNNING`` state.
+
+Shard Collections
+~~~~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster requires the 
+``sharding`` option, to specify how ``mongosync`` should shard collections.
+
+The ``sharding.shardEntries`` array specifies the collections to shard.
+Collections that are not listed in this array replicate as unsharded.
+
+Supporting Indexes
+~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster can automatically
+create a supporting index for the new shard key on the destination cluster.
+This is done by setting the ``sharding.createSupportingIndexes`` option.
+
+When ``sharding.createSupportingIndexes`` is ``false`` (the default): 
+
+* Each shard key you provide for the ``sharding.shardEntries`` option
+  must have an existing index on the source cluster.
+
+* One of the indexes used for the shard key must have simple collation if the 
+  collection uses any other collation.
+
+* To use a unique index in the shard key, you must specify its uniqueness when
+  you create the index on the source cluster. 
+
+* Unique indexes on the source cluster that are incompatible with the requested
+  shard key on the destination cluster, such as a unique index on the source
+  that does not contain the requested shard key as a prefix on the destination,
+  can cause ``mongosync`` to fail.
+
+When ``sharding.createSupportingIndexes`` is ``true``: 
+
+* ``mongosync`` synchronizes indexes on the source cluster to the destination.
+
+* If the supporting indexes exist on the source cluster, ``mongosync`` 
+  synchronizes the indexes to the destination cluster and uses them 
+  as the shard key.
+
+* If the supporting indexes don't exist, ``mongosync`` creates them on the
+  destination cluster.
+
+The ``sharding.createSupportingIndexes`` option affects all collections sharded
+during sync.
+
+
+Rename During Sync
+~~~~~~~~~~~~~~~~~~
+
+Collections synchronized from a replica set to a sharded cluster listed in
+the ``sharding.shardEntries`` array become sharded collections on the
+destination cluster.
```

Comment:
[question, minimalism]

Do we need to say "listed in the ``sharding.shardEntries`` array" in this sentence? The sentence makes sense to me without that phrase, so removing it might make this a bit clearer.

Reply:
I reworked it a bit, but I think we need the "listed" clause to drive home that collections not in that list remain unsharded.